Modules/@babylonjs/react-native/windows/BabylonReactNative/readme.txt

```diff
@@ -0,0 +1,23 @@
+========================================================================
```

Comment:
Seems like this file could be removed? Is it really relevant to this project?

Reply:
This file is auto-added when installing winrt c++. Will remove